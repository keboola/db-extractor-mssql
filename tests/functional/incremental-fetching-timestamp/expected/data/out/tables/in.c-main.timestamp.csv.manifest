{
    "destination": "in.c-main.timestamp",
    "incremental": true,
    "primary_key": [
        "id"
    ],
    "metadata": [
        {
            "key": "KBC.name",
            "value": "timestamp_test"
        },
        {
            "key": "KBC.sanitizedName",
            "value": "timestamp_test"
        },
        {
            "key": "KBC.schema",
            "value": "dbo"
        },
        {
            "key": "KBC.catalog",
            "value": "test"
        },
        {
            "key": "KBC.type",
            "value": "BASE TABLE"
        }
    ],
    "column_metadata": {
        "id": [
            {
                "key": "KBC.datatype.type",
                "value": "int"
            },
            {
                "key": "KBC.datatype.nullable",
                "value": false
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "INTEGER"
            },
            {
                "key": "KBC.datatype.length",
                "value": "10"
            },
            {
                "key": "KBC.sourceName",
                "value": "id"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "id"
            },
            {
                "key": "KBC.primaryKey",
                "value": false
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 1
            }
        ],
        "timestamp": [
            {
                "key": "KBC.datatype.type",
                "value": "timestamp"
            },
            {
                "key": "KBC.datatype.nullable",
                "value": false
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "STRING"
            },
            {
                "key": "KBC.sourceName",
                "value": "timestamp"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "timestamp"
            },
            {
                "key": "KBC.primaryKey",
                "value": false
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 2
            }
        ]
    },
    "columns": [
        "id",
        "timestamp"
    ]
}
