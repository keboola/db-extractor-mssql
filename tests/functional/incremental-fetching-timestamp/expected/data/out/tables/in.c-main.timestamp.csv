1,0x<<RANDOM>>
2,0x<<RANDOM>>
3,0x<<RANDOM>>
